api/ccapi/include/model.h

```diff
@@ -93,7 +93,9 @@ for inference and training without any configurations*/
     ML_TRAIN_MODEL_FORMAT_FLATBUFFER,             /**< flatbuffer file */
   MODEL_FORMAT_ONNX = ML_TRAIN_MODEL_FORMAT_ONNX, /**< ONNX file */
 
-  MODEL_FORMAT_QNN = ML_TRAIN_MODEL_FORMAT_QNN /**< qnn binary file */
+  MODEL_FORMAT_QNN = ML_TRAIN_MODEL_FORMAT_QNN, /**< qnn binary file */
+  MODEL_FORMAT_KVCACHE = ML_TRAIN_MODEL_FORMAT_KVCACHE,
+  MODEL_FORMAT_LORA_BIN = ML_TRAIN_MODEL_FORMAT_LORA_BIN
```

Comment:
I wonder do we need this?

Reply:
It was an unnecessary Enum, so I deleted it.